networking/networking_operators/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -23,10 +23,17 @@ For an overview of the AWS Load Balancer Operator, see xref:../../../networking/
 AWS Load Balancer Operator currently does not support AWS GovCloud.
 ====
 
+[id="aws-load-balancer-operator-release-notes-1.2"]
+== AWS Load Balancer Operator 1.2.0
```

Comment:
We should mention that this release supports the version 2.8.2 of the AWS Load Balancer Controller.